quickwit/rest-api-tests/scenarii/es_compatibility/0018-search_after.yaml

```diff
@@ -44,3 +44,102 @@ expected:
       relation: eq
     hits:
       - sort: [9018]
+---
+# Test with a search after value as string
+# Quickwit should convert it to the correct type
+json:
+  size: 1
+  query:
+      match_all: {}
+  sort:
+    - actor.id:
+        order: asc
+  search_after: ["5688"]
+expected:
+  hits:
+    total:
+      value: 100
+      relation: eq
+    hits:
+      - sort: [9018]
+---
+json:
+  size: 1
+  query:
+      match_all: {}
+  sort:
+    - actor.id:
+        order: asc
+  search_after: [5688]
+expected:
+  hits:
+    total:
+      value: 100
+      relation: eq
+    hits:
+      - sort: [9018]
+---
+json:
+  size: 100
+  track_total_hits: true
+  query:
+      match_all: {}
+  sort:
+    - created_at:
+        order: asc
+  search_after: [1422748815000000000]
+expected:
+  hits:
+    hits:
+      $expect: "len(val) == 4"
+---
+# Accept timestamp as string
+json:
+  size: 100
+  track_total_hits: true
+  query:
+      match_all: {}
+  sort:
+    - created_at:
+        order: asc
+  search_after: ["1422748815000000000"]
+expected:
+  hits:
+    hits:
+      $expect: "len(val) == 4"
+---
+# Accept timestamp in epoch_millis
+json:
+  size: 100
+  track_total_hits: true
+  query:
+      match_all: {}
+  sort:
+    - created_at:
+        order: asc
+        format: epoch_millis
```

Comment:
did you run this tests against elastic too? If they don't pass, please comment and add 

`engine: quickwit`

Reply:
good catch